### PR TITLE
Fix product _compute_quantities context dependencies

### PR DESCRIPTION
**Pull request by alessandro-fiorino**
Description of the issue/feature this PR addresses:
_compute_quantities calls _compute_quantities_dict which calls _get_domain_locations, this last function depends also on 'location' and 'warehouse' from context, so also this variables should be put in the depends_context decorator

Current behavior before PR:
product.qty_available and product.with_context(location=x).qty_available return the same value also if the product in stored in more than one location
Desired behavior after PR is merged:
product.with_context(location=x).qty_available should return the quantity available only in location x

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by ryv-odoo — APPROVED** [review]
LGTM.
It seems that the fix is already on next stable version.

@fw-bot  ignore

**Comment by alessandro-fiorino** [comment]
My CLA is waiting for approval here https://github.com/odoo/odoo/pull/69681

**Comment by alessandro-fiorino** [comment]
Wrong rebase will redo it

**Pull request closed without merging**
